### DISCUSSION TITLE
Can you reduce the amount of updates?

### DISCUSSION THREAD

**Discussion by ghost** [ORelio/Minecraft-Console-Client#2259] [General]
It seems like there is an update every day, and that shows that the devs are paying attention and are actively trying to improve it. 

Yet I feel like having an update every day is quite excessive. Would it be possible to compress the updates so that it updates every week - or even make it automatically update itself?

**Comment by milutinke** [comment]
You don't have to use the latest updates, just check once a week.
We have an automatic build system which compiles the project for each pull request merge / direct commit.
